### DISCUSSION TITLE
Oops... conditional access

### DISCUSSION THREAD

**Discussion by Doerka** [Micke-K/IntuneManagement#139] [Ideas]
Hi Micke,

Normally I export CA policies from a test-tenant where the policies are set to read-only. Because these are somewhat outdated, I had exported the policies that I configured last week at another customer. These policies were on... 
During the import I forgot to change the state from "As Exported" to "Report only" with all the consequences... calling Microsoft and little sleep. Is it an idea to change the default setting to "Off" or "Report-only" instead of "As Exported" to prevent this from happening to others.

**Comment by Micke-K** [comment]
Hello!

That doesn't sound good. I hope it's fixed quickly. 

I actually thought audit was default. I'll have a look because it is a very good point if it isn't.

Cheers!

**Reply by Doerka** [reply]
Microsoft's data protection team performs a very extensive verification check. That takes time. Especially if you cannot answer all questions (in time).

**Reply by Micke-K** [reply]
It wasn't set to Audit by default. 

I updated it so it will be in the next release. 

I'm so sorry that it caused lots of trouble for you!

Cheers!

**Reply by Micke-K** [reply]
I actually added a new option and made that default, 
As Exported - Change On to Report-only

This won't change disabled policies but will make sure that no policy is imported as On. 

Cheers!

**Reply by Doerka** [reply]
Thanks Micke, Your tool also saved me a lot of time. Let's see it as a learning opportunity. The entire process with Microsoft has also brought to light several things that can help us next time.
